test/src/ariaqueryhandler.spec.ts

```diff
@@ -581,7 +577,7 @@ describe('AriaQueryHandler', () => {
       await page.waitForSelector('aria/zombo', {timeout: 10}).catch(error_ => {
         return (error = error_);
       });
-      expect(error!.stack).toContain('waiting for selector `zombo` failed');
+      expect(error!.stack).toContain('Waiting failed: 10ms exceeded');
```

Comment:
Can we still have the details error messages?

Reply:
I think the line does not necessarily tells which selector failed (selector might be computed or received from an external system). I think we should keep these details in the error message: they seem quite useful.